src/components/Reports/GoalCalculator/Shared/GoalCalculatorSection.tsx

```diff
@@ -30,31 +32,49 @@ export const GoalCalculatorSection: React.FC<GoalCalculatorSectionProps> = ({
   return (
     <div>
       <Box pb={4}>
-        <Stack direction="row" justifyContent="space-between">
-          <Typography variant="h6">
-            {title}
-            {rightPanelContent && (
-              <IconButton
+        <Stack direction="row" spacing={2}>
+          <Stack
```

Comment:
Do we need the nested Stacks? It puts the button group far away from the title. I think a single `<Stack direction="row" gap={2}>` Stack works.